template.md

```diff
@@ -0,0 +1,321 @@
+- **ms.devlang** defines the technology. Some of the supported values are: dotnet, cpp, csharp, fsharp, xml, vb and xml.
+- **ms.assetid**: This is the GUID of the article that is used for internal tracking purposes such as Business Intelligence (BI). When creating a new Markdown file, get a GUID from [https://www.guidgenerator.com](https://www.guidgenerator.com). 
+
+## Basic Markdown, GFM, and special characters
+
+All basic and GitHub-flavored Markdown (GFM) is supported. For more information on these, see:
+
+- [Baseline Markdown syntax](https://daringfireball.net/projects/markdown/syntax)
+- [GitHub-flavored Markdown documentation](https://guides.github.com/features/mastering-markdown)
+
+Markdown uses special characters such as \*, \`, and \# for formatting. If you wish to include one of these characters in your content, you must do one of two things:
+
+- Put a backslash before the special character to "escape" it (for example, `\*` for a \*)
+- Use the [HTML entity code](http://www.ascii.cl/htmlcodes.htm) for the character (for example, `&#42;` for a &#42;).
+
+## File name
+
+File names use the following rules:
+* Contain only lowercase letters, numbers, and hyphens.
+* No spaces or punctuation characters. Use the hyphens to separate words and numbers in the file name.
+* Use action verbs that are specific, such as develop, buy, build, troubleshoot. No -ing words.
+* No small words - don't include a, and, the, in, or, etc.
+* Must be in Markdown and use the .md file extension.
+
+## Headings
+
+Use sentence-style capitalization. Always capitalize:
+- The first word of a heading. 
+- The word following a colon in a title or heading (for example, "How to: Sort an array"). 
+
+Headings should be done using atx-style, that is, use 1-6 hash characters (#) at the start of the line to indicate a heading, corresponding to HTML headings levels H1 through H6. Examples of first- and second-level headers are used above. 
+
+There **must** be only one first-level heading (H1) in your topic, which will be displayed as the on-page title.
+
+If your heading finishes with a `#` character, you need to add an extra `#` character in the end in order for the title to render correctly. For example, `# Async Programming in F# #`.     
+
+Second-level headings will generate the on-page TOC that appears in the "In this article" section underneath the on-page title.
+
+### Third-level heading
+#### Fourth-level heading
+##### Fifth level heading
+###### Sixth-level heading
+ 
+## Text styling
+
+*Italics*
+ Use for files, folders, paths (for long items, split onto their own line) - new terms - URLs (unless rendered as links, which is the default).
+
+**Bold**
+Use for UI elements.
+
+~~Strikethrough~~
+
+## Links
+
+### Internal Links
+
+To link to a header in the same Markdown file, view the source of the published article, find the id of the head (for example, `id="blockquote"`), and link using # + id (for example, `#blockquote`).
```

Comment:
Can we mention that view source in published doc may not be necessary, given the anchors are auto-generated from the headings? 
If we keep this as is, "find the id of the head" -->"find the id of the heading"